src/cpu.rs

```diff
@@ -1336,9 +1435,9 @@ mod tests {
     #[cfg(test)]
     fn compare_test_helper<F>(compare: &mut F, load_instruction: Instruction)
     where
-        F: FnMut(&mut CPU, u8),
+        F: FnMut(&mut CPU<crate::memory::Memory>, u8),
```

Comment:
We use `crate::memory::Memory` quite a lot. Why not import it?
```rust
use crate::memory::Memory
```
or, if we want to be more explicit
```rust
use crate::memory::Memory as Ram
```